docs/interactions/Receiving_and_Responding.md

```diff
@@ -274,20 +274,29 @@ Every Interaction is sent with the following headers:
 Using your favorite security library, you **must validate the request each time you receive an [interaction](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object)**. If the signature fails validation, respond with a `401` error code. Here's a couple code examples:
 
 ```js
-const nacl = require('tweetnacl');
+const crypto = require('crypto');
 
 // Your public key can be found on your application in the Developer Portal
 const PUBLIC_KEY = 'APPLICATION_PUBLIC_KEY';
 
+const IMPORTED_PUBLIC_KEY=await crypto.webcrypto.subtle.importKey(
+  "raw",
+  Buffer.from(process.env.PUBLIC_KEY, "hex"),
+  { name: "NODE-ED25519", namedCurve: "NODE-ED25519", public: true },
+  false,
+  ["verify"]) 
+
```

Comment:
```suggestion
const IMPORTED_PUBLIC_KEY = await crypto.webcrypto.subtle.importKey(
  "raw",
  Buffer.from(process.env.PUBLIC_KEY, "hex"),
  { name: "NODE-ED25519", namedCurve: "NODE-ED25519", public: true },
  false,
  ["verify"]
);
```

Reply:
Thanks for formatting, that's better now!